pkg/util/rand/rand.go

```diff
@@ -32,19 +32,13 @@ var rng = struct {
 	rand: rand.New(rand.NewSource(time.Now().UTC().UnixNano())),
 }
 
-// String generates a random alphanumeric string n characters long.  This will
-// panic if n is less than zero.
-func String(n int) string {
-	if n < 0 {
-		panic("out-of-bounds value")
-	}
-	b := make([]rune, n)
+// Int generates an integer in range 0->max.
+// By design this should panic if input is invalid, <= 0.
+func Int(max int) int {
```

Comment:
This hunk is reordering the `String(n)` function to the bottom of the file, so that the stuff using locks on `rng` and so on are all in the same place.